test/utils.js

```diff
@@ -0,0 +1,48 @@
+var assert = require('chai').assert;
+var sass = require('node-sass');
+
+var render = function(options) {
+  options.outputStyle = 'compressed';
```

Comment:
Any reason you hard-code this rather than respecting possibly given `outputStyle`?